### PR TITLE
CORS Concept Used

### PR DESCRIPTION
**Pull request by rajuTK9**

### CODE REVIEW

File: client/src/components/Index/IndexPage.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ export default function IndexPage() {
                   return (
```
Comment (aditya-190) [inline client/src/components/Index/IndexPage.js]: The logic can be improved a bit here using ternary operator, try optimising it using only one return statement.

File: client/src/components/Learning/Content/Content.js [hunk 2]
```diff
@@ -12,7 +12,10 @@ export default function Content(props) {
       setChapterIndex(chapters);
       e.setAttribute("id", e.innerText.split(" ").join(""));
     });
-  }, [setChapterIndex]);
+
+    if (headings.length < 1) setChapterIndex([]);
+    console.log("first");
```
Comment (aditya-190) [inline client/src/components/Learning/Content/Content.js]: Please remove this console.

**Review by aditya-190 — APPROVED** [review]
Pls make the suggested changes in your next PR's. Looks good to me.

File: client/src/components/Learning/Content/Content.js (hunk 2 above)
Comment (rajuTK9) [reply client/src/components/Learning/Content/Content.js]: Oh sorry, I forgot to remove it! 😅 Will fix it in next commit

Comment (aditya-190) [reply client/src/components/Learning/Content/Content.js]: Sure.

File: client/src/components/Index/IndexPage.js (hunk 1 above)
Comment (rajuTK9) [reply client/src/components/Index/IndexPage.js]: Okay, it will be improved in the next PR.
Thanks for pointing out.

**Review by aditya-190 — APPROVED** [review]
Looks good to me. Pls merge the changes now.

**Pull request merged**